packages/firestore/src/core/firestore_client.ts

```diff
@@ -507,9 +508,31 @@ export function firestoreClientRunAggregationQuery(
   client: FirestoreClient,
   query: AggregateQuery
 ): Promise<AggregateQuerySnapshot> {
-  return client.asyncQueue.enqueue(() => {
```

Comment:
same function updated in the PR: https://github.com/firebase/firebase-js-sdk/pull/6579/files . copied here to make sure tests will pass

Reply:
Please make sure to merge the other PR first then.